pkgs/tools/networking/pmacct/default.nix

```diff
@@ -1,55 +1,66 @@
-{ lib, stdenv
+{ lib
+, stdenv
 , fetchFromGitHub
 , pkg-config
 , autoreconfHook
 , libtool
 , libpcap
+, libcdada
 
-# Optional Dependencies
-, zlib ? null
-, withJansson ? true, jansson ? null
-, withNflog ? true, libnetfilter_log ? null
-, withSQLite ? true, sqlite ? null
-, withPgSQL ? true, postgresql ? null
-, withMysql ? true, libmysqlclient ? null }:
-
-assert withJansson -> jansson != null;
-assert withNflog -> libnetfilter_log != null;
-assert withSQLite -> sqlite != null;
-assert withPgSQL -> postgresql != null;
-assert withMysql -> libmysqlclient != null;
-
-let inherit (lib) getDev optional optionalString; in
+  # Optional Dependencies
+, withJansson ? true, jansson
+, withNflog ? true, libnetfilter_log
+, withSQLite ? true, sqlite
+, withPgSQL ? true, postgresql
+, withMysql ? true, libmysqlclient, zlib
+, gnutlsSupport ? false, gnutls
+}:
 
 stdenv.mkDerivation rec {
-  version = "1.7.5";
+  version = "1.7.6";
   pname = "pmacct";
 
   src = fetchFromGitHub {
     owner = "pmacct";
-    repo = pname;
-    rev = "v${version}";
-    sha256 = "17p5isrq5w58hvmzhc6akbd37ins3c95g0rvhhdm0v33khzxmran";
+    repo = "pmacct";
+    rev = "${version}";
+    sha256 = "0x1i75hwz44siqvn4i58jgji0zwrqgn6ayv89s9m9nh3b423nsiv";
   };
 
-  nativeBuildInputs = [ autoreconfHook pkg-config libtool ];
-  buildInputs = [ libpcap ]
-    ++ optional withJansson jansson
-    ++ optional withNflog libnetfilter_log
-    ++ optional withSQLite sqlite
-    ++ optional withPgSQL postgresql
-    ++ optional withMysql [ libmysqlclient zlib ];
+  nativeBuildInputs = [
+    autoreconfHook
+    pkg-config
+    libtool
+  ];
+  buildInputs = [
+    libcdada
+    libpcap
+  ] ++ lib.optionals withJansson [
+    jansson
+  ] ++ lib.optionals withNflog [
+    libnetfilter_log
+  ] ++ lib.optionals withSQLite [
+    sqlite
+  ] ++ lib.optionals withPgSQL [
+    postgresql
+  ] ++ lib.optionals withMysql [
+    libmysqlclient
+    zlib
+  ] ++ lib.optionals gnutlsSupport [
+    gnutls
+  ];
```

Comment:
I think the way it was before with lib.optional was easier to read and it is actual less code.